### ISSUE TITLE
boombox turning off

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#13932]
(WEB REPORT BY: edwarus REMOTE: 172.93.109.202:7777)
>Revisiond 00c2630c3a91e416e5dd79dcb7da69b164ce4e9 

>General description of the issue: Turning off

>What you expected to happen: walk through the station pumpin my music

>What actually happened: It turns off as you go through new new zones

>Steps to reproduce if possible: bring the boombox around the station while its playing

**Comment by jknpj** [comment]
Boomboxes are literally just portable sound systems so yeah they get fucky when you go to places that already have a sound system or jukebox.

**Comment by Exxion** [comment]
The problem is probably that they have no fucking idea how to deal with changing areas, because whoever coded them somehow didn't think about someone actually moving the portable sound system

**Comment by FudgePucker** [comment]
Well the music is played in the area it is in, so changing areas would obviously fuck everything up.

**Comment by Exxion** [comment]
Yeah but they could have actually taken into account that it might switch areas and
You know
Update the area the music is played in

**Comment by FudgePucker** [comment]
Wouldn't it need to check area every tick then?

**Comment by Exxion** [comment]
No, it would need to check the area when moved.
